x-pack/solutions/observability/plugins/ux/dev_docs/testplan.md

```diff
@@ -0,0 +1,22 @@
+# Manual test plan
```

Comment:
nit: changing filename `testplan` -> `test_plan` to match [navigation_test_plan](https://github.com/elastic/kibana/pull/212146)